src/main/java/Main.java

```diff
@@ -41,128 +41,131 @@
             System.out.println("Przed partią - wpisz 'przed', Po partii - wpisz 'po'");
-            etap = input.next();
+            stageOfTheMatch = input.next();
         }
 
 
     }
 
 
-    private int obliczWartoscReki() {
+    private int calculateValueOfHand() {
 
         int j;
-        if (etap.toLowerCase().equals("przed")) j = 1;
+        if (stageOfTheMatch.toLowerCase().equals("przed")) j = 1;
         else j = 2;
 
-        if (wartoscReki <= PRZELOMOWA_WARTOSC_REKI) return tab2[wartoscReki - MIN_WARTOSC_REKI][j];
-        else return tab2[tab2.length - 1][j];
+        if (handValue <= DISRUPTIVE_HAND_VALUE)
+            return tableOFPossibleHandValuesBeforeAndAfterMatch[handValue - MIN_HAND_VALUE][j];
+        else
+            return tableOFPossibleHandValuesBeforeAndAfterMatch[tableOFPossibleHandValuesBeforeAndAfterMatch.length - 1][j];
 
     }
 
 
-    private void pobierzWynikiPartii() {
+    private void getMatchResults() {
 
         Scanner s = new Scanner(System.in);
         System.out.println("Ilosc wylicytowanych lew: ");
-        iloscWylicytowanychLew = s.nextInt();
-        if ((iloscWylicytowanychLew < MIN_ILOSC_LEW) || (iloscWylicytowanychLew > MAX_ILOSC_LEW)) {
+        amountOfBidedTricks = s.nextInt();
+        if ((amountOfBidedTricks < MIN_AMOUNT_OF_TRICKS) || (amountOfBidedTricks > MAX_AMOUNT_OF_TRICKS)) {
             throw new RuntimeException("Podano niepoprawna ilosc wylicytowanych lew");
         }
 
         System.out.println("Podaj kolor: ");
-        String color;
-        color = s.next();
+        String colour;
+        colour = s.next();
 
-        switch (color.toLowerCase()) {
+        switch (colour.toLowerCase()) {
             case "trefl":
-                kolor = Kolor.TREFL;
+                suit = Suit.TREFL;
                 break;
             case "karo":
-                kolor = Kolor.KARO;
+                suit = Suit.KARO;
                 break;
             case "kier":
-                kolor = Kolor.KIER;
+                suit = Suit.KIER;
                 break;
             case "pik":
-                kolor = Kolor.PIK;
+                suit = Suit.PIK;
                 break;
             case "ba":
-                kolor = Kolor.BA;
+                suit = Suit.BA;
                 break;
             default:
                 throw new RuntimeException("Niepoprawny kolor!");
 
         }
 
         System.out.println("Ilosc wpadek: ");
-        iloscWpadek = s.nextInt();
-        if (iloscWpadek == 0) {
+        amountOfBloomers = s.nextInt();
+        if (amountOfBloomers == 0) {
             System.out.println("Ilosc nadrobek:");
-            iloscNadrobek = s.nextInt();
+            amountOfOvertricks = s.nextInt();
         }
 
 
         System.out.println("Czy byla kontra? t-tak, n-nie");
-        kontra = s.next().charAt(0);
+        versus = s.next().charAt(0);
```

Comment:
Veto jest lepszym tłumaczeniem. Versus używa się raczej jako "przeciwko". Ew. zawsze możesz zostać przy counter.

Reply:
a rekontra to będzie reveto? czy niekoniecznie?